### PR TITLE
feat(poi): address displaying was added

### LINKED ISSUE #45: Show address of poi on card

### PR DESCRIPTION
**Pull request by leushkin**
closes #45

### CODE REVIEW

**Comment by lxndr** [comment]
we've also got 'building'

**Comment by leushkin** [comment]
should I create utility function which takes street obj, building and returns address string?

**Comment by lxndr** [comment]
that would be nice

File: client/src/utils.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import _ from 'lodash';
+import * as Types from './types/graphql';
+
+export const createAddress = ({street, building}: Pick<Types.Poi, 'street' | 'building'>) => {
```
Comment (lxndr) [inline client/src/utils.ts]: formatAddress?

Comment (leushkin) [reply client/src/utils.ts]: > formatAddress?

done

**Pull request merged**